src/notification.rs

```diff
@@ -252,6 +256,24 @@ impl Notification for PublishDiagnostics {
     const METHOD: &'static str = "textDocument/publishDiagnostics";
 }
 
+#[cfg(feature = "proposed")]
+pub use proposed::*;
+
+#[cfg(feature = "proposed")]
+pub mod proposed {
+    use super::*;
+
+    /// The progress notification is sent from the server to the client to ask
+    /// the client to indicate progress.
+    #[derive(Debug)]
+    pub enum Progress {}
```

Comment:
it's weird, although I'm doing `pub use proposed::*;` in both `src/lib.rs` and `src/notification.rs` I can refer to
```rust
lsp_types::ProgressParams
```
but not
```rust
lsp_types::notification::Progress // needs `proposed::Progress`
```
any ideas why/how to solve that? Do we want to retain the explicit `proposed` mod?

Reply:
No clue.

Would be nice to have it consistently exported though. If the module needs to be removed for that then removing it seems like the better option.

Reply:
Removed the module then